apps/website-25/src/components/homepage/CommunitySection/index.tsx

```diff
@@ -6,7 +6,7 @@ import CommunityValuesSection from './CommunityValuesSection';
 
 const CommunitySection = () => {
   return (
-    <Section title="Our community" className="community-section">
+    <Section className="community-section">
```

Comment:
Title moved into `CommunityValuesSection` to allow the buttons to live in the header